src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskBySearchCriteriaPactTest.java

```diff
@@ -86,51 +89,53 @@ public void getTasksBySearchCriteria() {
-            "Taylor House",
-            "Asylum",
-            "1617708245308495",
-            "refusalOfHumanRights",
-            "John Doe",
-            true);
+        var tasks = new ArrayList<Task>();
+        var taskOne =  new Task("4d4b6fgh-c91f-433f-92ac-e456ae34f72a",
+                                              "Jake",
+                                              "ReviewTheAppeal",
+                                              "unconfigured",
+                                              "SELF",
+                                              "PRIVATE",
+                                              "task name",
+                                              ZonedDateTime.now(),
+                                              ZonedDateTime.now(),
+                                              "Mark Alistair",
+                                              true,
+                                              "Time extension",
+                                              "IA",
+                                              "1",
+                                              "765324",
+                                              "Newcastle",
+                                              "Asylum",
+                                              "4d4b3a4e-c91f-433f-92ac-e456ae34f72a",
+                                              "processApplication",
+                                              "Bob Smith",
+                                              true,
+                                                new WarningValues(Collections.emptyList())
+        );
+
+        var taskTwo =  new Task("4d4b6fgh-cc1f-433f-92ac-e456aed4f72a",
+                                              "Megan",
+                                              "ReviewTheAppeal",
+                                              "unconfigured",
+                                              "SELF",
+                                              "PRIVATE",
+                                              "task name",
+                                              ZonedDateTime.now(),
+                                              ZonedDateTime.now(),
+                                              "Jean Pierre",
+                                              true,
+                                              "Time extension",
+                                              "IA",
+                                              "1",
+                                              "766524",
+                                              "Newcastle",
+                                              "Asylum",
+                                              "4d4b3a4e-c9df-43sf-92ac-e456ee34fe2a",
+                                              "processApplication",
+                                              "Bob Smith",
+                                              true,
+                                              new WarningValues(Collections.emptyList()));
```

Comment:
Can we revert the pact test values these don't look like the correct ones e.g asignee should be a UUID etc..